### ISSUE TITLE
is dead the project?

### ISSUE THREAD

**Issue by Allan-Nava** [pedroSG94/Stream-USB-test#7]

**Comment by pedroSG94** [comment]
Yes, the reason is that I haven't usb camera anymore and usb library is also abandoned.

**Comment by Allan-Nava** [comment]
No wayyy, i'm trying to develop something with usb camera

**Comment by pedroSG94** [comment]
Sorry, but I don' want develop using a project with 3 years without updates. Fix possible errors of USB library without any help is a pain and it is developed in C++ that is not my main language.
I have no reason to buy a camera and maintain this since USB library is abandoned.

**Comment by Allan-Nava** [comment]
Ok :(
